hathor/manager.py

```diff
@@ -419,27 +419,22 @@ def _initialize_components(self) -> None:
             try:
                 if self._full_verification:
                     # TODO: deal with invalid tx
-                    if self.tx_storage.is_tx_needed(tx.hash):
-                        assert isinstance(tx, Transaction)
-                        tx._height_cache = self.tx_storage.needed_index_height(tx.hash)
```

Comment:
Why don't we need it anymore?

Reply:
Also because `tx._height_cache` not being in use anymore.

Reply:
Same as in the other conversation thread, right? Thanks!